YMChat/index.html

```diff
@@ -72,6 +73,7 @@
             pwa: true,
             sdk: true,
             ymAuthenticationToken: ymAuthenticationToken,
+            setDisableActionsTimeout: disableActionsOnLoad == 'true',
```

Comment:
Just check with sukrit if we have to send the parameter when value is false

Reply:
Sukrit and I had a discussion. It's okay to set the value as false if the `disableActionsOnLoad` parameter isn't set.